lib/itamae/backend.rb

```diff
@@ -283,6 +283,9 @@ def ssh_options
           opts.merge!(password: password)
         end
 
+        opts[:verify_host_key] = :never if opts[:strict_host_key_checking] == false
```

Comment:
nit
```
opts[:verify_host_key] = :never unless opts[:strict_host_key_checking]
```

Reply:
I wonder if opts[:strict_host_key_checking] might return `nil` if target ssh_config didn't have `StrictHostKeyChecking` configuration. I think it would lead to unexpected behavior for users.

ref: https://github.com/net-ssh/net-ssh/blob/139126b08ded6fe2451c3791a1c041f378920a05/lib/net/ssh/config.rb#L219

Reply:
I got it. Thanks.